framework/web/XmlResponseFormatter.php

```diff
@@ -142,4 +143,28 @@ protected function formatScalarValue($value)
 
         return (string) $value;
     }
+
+    /**
+     * @param mixed $name
+     * @return string
+     */
+    protected function getElementName($name)
```

Comment:
the method name could be made more precise regarding the contents of the method.